### ISSUE TITLE
Bundle PhoneGap 1.3.0 and jQuery Mobile 1.0

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#49]
```
Or newer 

```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 17 Nov 2011 at 10:19

**Comment by GoogleCodeExporter** [comment]
```
PhoneGap 1.3.0 released December 19, 2011
```

Original comment by `paul.beu...@gmail.com` on 20 Dec 2011 at 12:47
- Changed title: **Bundle PhoneGap 1.3.0 and jQuery Mobile 1.0**
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixes available with 1.2.6
```

Original comment by `paul.beu...@gmail.com` on 21 Dec 2011 at 3:57
- Changed state: **Fixed**
